class.jetpack-jitm.php

```diff
@@ -414,6 +426,60 @@ function videopress_media_upload_warning_msg() {
 		<?php
 	}
 
+	/**
+	 * Display message prompting user to install WooCommerce Services.
+	 *
+	 * @since 4.6
+	 */
+	function woocommerce_services_msg() {
+		if ( ! current_user_can( 'manage_woocommerce' ) || ! current_user_can( 'install_plugins' ) ) {
+			return;
+		}
+
+		if ( isset( self::$jetpack_hide_jitm['woocommerce_services'] ) ) {
+			return;
+		}
+
+		if ( ! function_exists( 'wc_get_base_location' ) ) {
+			return;
+		}
+
+		$base_location = wc_get_base_location();
+
+		switch ( $base_location['country'] ) {
+			case 'US':
+				$message = __( 'Try our new service for USPS shipping & label-printing.', 'jetpack' );
+				break;
+			case 'CA':
+				$message = __( 'Try our new Canada Post shipping service.', 'jetpack' );
+				break;
+			default:
+				return;
+		}
+
+		$install_url = wp_nonce_url( add_query_arg( array( 'wc-services-action' => 'install' ) ), 'wc-services-install' );
+
+		?>
+		<div class="jp-jitm woo-jitm">
+			<a href="#"  data-module="woocommerce_services" class="dismiss"><span class="genericon genericon-close"></span></a>
+			<div class="jp-emblem">
+				<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="0" y="0" viewBox="0 0 24 24" enable-background="new 0 0 24 24" xml:space="preserve">
+					<path d="M18,8h-2V7c0-1.105-0.895-2-2-2H4C2.895,5,2,5.895,2,7v10h2c0,1.657,1.343,3,3,3s3-1.343,3-3h4c0,1.657,1.343,3,3,3s3-1.343,3-3h2v-5L18,8z M7,18.5c-0.828,0-1.5-0.672-1.5-1.5s0.672-1.5,1.5-1.5s1.5,0.672,1.5,1.5S7.828,18.5,7,18.5z M4,14V7h10v7H4z M17,18.5c-0.828,0-1.5-0.672-1.5-1.5s0.672-1.5,1.5-1.5s1.5,0.672,1.5,1.5S17.828,18.5,17,18.5z" />
+				</svg>
+			</div>
+			<p class="msg">
```

Comment:
It's not the most straightforward logic perhaps, but if you don't fall into either the `US` or `CA` cases, the `default` simply returns. See line [457](https://github.com/Automattic/jetpack/pull/6183/files#diff-32dcaecae9168fb9b58487a1541dd587R457).